### DISCUSSION TITLE
Drawing the trajectory path

### DISCUSSION THREAD

**Discussion by camillerievan** [zapatacomputing/orqviz#62] [General]
I managed to draw the trajectory path of my minimization function on the 2D graph - how can I do the same on the 3D graph?

**Comment by MSRudolph** [comment]
Hey @camillerievan, you can try to get the 2D trajectory as usual, and then create a new list with the corresponding loss values, lets call them `z`. You should be able to plot the line via something like
`ax.plot(x, y, zs=z)`
where `ax` is your 3D axis, `x` and `y` are your 2D projection coordinates.

In short, you can plot on a 3D surface just like you would on any matplotlib 3D axis.
